web/src/constants/Theme.constants.ts

```diff
@@ -25,6 +25,7 @@ export const theme: DefaultTheme = {
     md: '14px',
     lg: '16px',
     xl: '18px',
+    xxxl: '24px',
```

Comment:
`xxxl`? 👀

Reply:
I don't know man, 20 should be xxl si I just went for it 😂